### ISSUE TITLE
Structure of accessible fieldsets

### ISSUE THREAD

**Issue by jessegreenberg** [phetsims/balloons-and-static-electricity#102]
A prototype DOM representation of RadioButtonGroups looked like the following:

```html
<fieldset id="charge-view-widget" role="radiogroup" aria-describedby="cs-label cs-description">
    <legend id="cs-label">Charge Settings</legend>		  
        <label for="show-charges">Show all charges</label>
        <input id="show-charges" name="charge-settings" value="show-all" type="radio" role="radio" aria-checked="true">
				
     <label for="show-no-charges">Show no charges</label>
     <input id="show-no-charges" name="charge-settings" value="show-none" type="radio" role="radio" aria-checked="false">
					

     <label for="show-charge-diffs">Show charge differences</label>
     <input id="show-charge-diffs" name="charge-settings" value="show-differences" type="radio" role="radio" aria-checked="false"> 								
     <p id="cs-description">Select how you want to see or hear the charges.</p>
</fieldset>
```

This issue is in balloons-and-static-electricity as that is the sim being used to experiment with accessibility.

The browser tends to 'combine' fieldsets, so that if there is more than one fieldset, the user will get loop through all inputs of every fieldset in the document instead of just the representative element.  The user is unable to tab navigate to various radio button groups.

To solve this, the following HTML structure seems to work nicely:
```html
<fieldset class="radiogroup"> 
	<legend>Select pizza crust</legend> 
	<ul class="radio"> 
		<li>
			<input type="radio" name="crust" id="crust1" value="deep" />
			<label for="crust1">Deep dish</label>
		</li> 
		<li><input type="radio" name="crust" id="crust2" value="thick" /><label for="crust2">Thick</label></li> 
		<li><input type="radio" name="crust" id="crust3" value="hand" /><label for="crust3">Hand thrown</label></li> 
		<li><input type="radio" name="crust" id="crust4" value="thin" /><label for="crust4">Thin</label></li> 
	</ul> 
</fieldset> 
```

This structure is a challenge to implement, it would not come naturally from the scene graph.

**Comment by jessegreenberg** [comment]
Scenery *could* be used to structure this, but it would involve creating new nodes just for accessibility structure and placing the RadioButtonGroupNodes under those as children.

A parent node could exist with accessible content containing the Fieldset and legend.
The RadioButtonGroup could exist as the un-ordered list, as a child of some AccessibleFieldsetNode

**Comment by jessegreenberg** [comment]
Though the above is possible, and was done, it did not fix the problem in the case of the sim.

**Comment by jessegreenberg** [comment]
Much ado about nothing.  Apparently, the ```name``` attribute is **required** for input elements of this type.  Web docs say that it can be used in a JavaScript or can be referenced in a form submission.  But it seems absolutely required for accessibility to correctly navigate the input elements.  Providing unique names solves this issue.  Committing shortly.

**Comment by jessegreenberg** [comment]
Accessible radio buttons now have a name attribute associated with them.  Closing.
